ui/components/app/wallet-overview/index.scss

```diff
@@ -73,7 +73,7 @@
   &__cached-secondary-balance {
     @include Paragraph;
 
-    color: var(--color-secondary-muted);
+    color: var(--color-warning-muted);
   }
 
   &__secondary-balance {
```

Comment:
These are different colors but updating to use the correct color semantically and secondary is deprecated

<img width="333" alt="Screen Shot 2022-08-11 at 4 15 43 PM" src="https://user-images.githubusercontent.com/8112138/184257867-e219837d-0ed6-4644-af59-8becb8116105.png"><img width="328" alt="Screen Shot 2022-08-11 at 4 15 52 PM" src="https://user-images.githubusercontent.com/8112138/184257868-ab228545-0c86-47cc-913e-c3339ad369a5.png">